assets/stylesheets/colors.scss

```diff
@@ -1,14 +1,16 @@
+@use "sass:color";
```

Comment:
@davidtaylorhq this fixes the issue from the previous PR

I also added a "compile color scheme stylesheet" step to the scss deprecation checker